crates/aranya-daemon/src/api.rs

```diff
@@ -413,6 +514,38 @@ impl DaemonApi for Api {
         Ok(())
     }
 
+    #[instrument(skip(self), err)]
+    async fn sync_hello_subscribe(
+        self,
+        _: context::Context,
+        peer: Addr,
+        team: api::TeamId,
+        delay: Duration,
+        duration: Duration,
+    ) -> api::Result<()> {
+        self.check_team_valid(team).await?;
+
+        self.peers
```

Comment:
We should change the name of this field to something more general. Such as `sync` or `syncer`. It is no longer just for configuring sync peers since it allows the user to subscribe to hello sync messages.
```suggestion
        self.sync
```